### PR TITLE
[Routing] fix docblock description for the build() method

### PR DESCRIPTION
**Pull request by xabbuh**
| Q | A |
| --- | --- |
| Fixed tickets | #15778 |
| License | MIT |

### CODE REVIEW

File: src/Symfony/Component/Routing/RouteCollectionBuilder.php [hunk 1]
```diff
@@ -238,7 +238,7 @@ public function setMethods($methods)
     }
 
     /**
-     * Creates the final ArrayCollection, returns it, and clears everything.
+     * Creates the final RouteCollection, returns it, and clears everything.
```
Comment (Tobion) [inline src/Symfony/Component/Routing/RouteCollectionBuilder.php]: what does "clears everything" mean? Doesnt look like it.

Comment (fabpot) [inline src/Symfony/Component/Routing/RouteCollectionBuilder.php]: indeed, that part should probably be removed.

Comment (xabbuh) [inline src/Symfony/Component/Routing/RouteCollectionBuilder.php]: done

**Comment by fabpot** [comment]
Thank you @xabbuh.

**Pull request merged**